content/blog/AML_projection.md

```diff
@@ -0,0 +1,68 @@
+\
+\
+{{% figure src="/blog_img/2019-06-14/workflow_partial1.PNG" width="60%" height="60%" %}}
+\
+\
+We have to load the data from the healthy individual and from the AML patient before the chemotherapy separately. Then run both datasets through the Genes widget, normalise it on counts per million and put it on a logarithmic scale in the Single Cell Preprocess widget. Now we make our primary t-SNE projection from the data of the healthy individual.
+\
+\
+{{% figure src="/blog_img/2019-06-14/workflow_partial2.PNG" width="70%" height="70%" %}}
+\
+\
+Next, we use the t-SNE projection of the cells from the healthy individual as a template for assessing the cells from the AML patient by using the Apply domain widget.
+<!--ali obstaja kakšna bolj primerna beseda kot assess v tem primeru?-->
+\
+\
+{{% figure src="/blog_img/2019-06-14/workflow.PNG" width="95%" height="95%" %}}
+\
+\
+Before we display the data, we have to merge it using the Concatenate widget where we select the append data source ID so that we can display this selection in the Scatter Plot widget. t-SNE is our primary data input and Apply Domain our template data input. 
+<br>
+
+Because we don’t want to reexecute t-SNE and run into problems with the batch effect in doing so, we use the Scatter Plot widget to display the data and set t-SNE-x and t-SNE-x as variables for the x and y axis. 
+\
+\
+{{% figure src="/blog_img/2019-06-14/Day0.png" width="95%" height="95%" %}}
+\
+\
+This is our projection of the single cell data from the AML patient before the chemotherapy onto t-SNE of the cells from a healthy person. By using the dataset sampled 15<sup>th</sup> day after the patient has first undergone chemotherapy, we can explore how the therapy affects cell populations.   
+\
+\
+{{% figure src="/blog_img/2019-06-14/Day15.png" width="95%" height="95%" %}}
+\
+\
+Looking at the clustering pattern from our earlier <a href= ”https://singlecell.biolab.si/blog/aml_identification/”>blog</a>, we can conclude that the cells here are predominantly T and natural killer cells, which is consistent with clearance of the cells during the chemotherapy.
+<br>
+
+And how about a month after the chemotherapy?
+\
+\
+{{% figure src="/blog_img/2019-06-14/Day31.png" width="95%" height="95%" %}}
+\
+\
+The hematopoietic stem cells have repopulated and, as we can see from the distribution of the grey points on the t-SNE representing cells from the patient, formed different cell types. 
+<br>
+<!--dodaj zaključek-->
```

Comment:
Short conclusion says that we have shown how to infer cell types with the help of t-SNE and a pre-existing, reference data set with known cell types.